packages/vuetify/src/components/VSheet/VSheet.tsx

```diff
@@ -19,7 +19,10 @@ export default defineComponent({
   name: 'VSheet',
 
   props: makeProps({
-    color: String,
+    color: {
```

Comment:
remove, if sheet doesn't have a bg that's a bug

Reply:
It is a bug, but dialog uses sheet for its content so this was needed.